k8s/demo/redis/redis-standalone.yml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis-standalone
+  labels:
+    app: redis-standalone
+spec:
+  ports:
```

Comment:
Since we are creating StatefulSets, add clusterIP: None and make this service a Headless Service.